etc/microfw.service

```
@@ -4,8 +4,7 @@ After=network.target docker.service
 
 [Service]
 Type=oneshot
-ExecStartPre=/usr/local/sbin/microfw tear_down
-ExecStart=/var/lib/microfw/setup.sh
+ExecStart=/usr/local/sbin/microfw apply_bootup
 RemainAfterExit=true
 ExecStop=/usr/local/sbin/microfw tear_down
 StandardOutput=journal
```

I just googled if `/dev/shm` is actually the right place to use for this, and got [sent to /run instead](https://wiki.debian.org/ReleaseGoals/%20RunDirectory#Overview). If we add a `RuntimeDirectory` here like so, systemd will create a `/run/microfw`  for us to use:

```suggestion
StandardOutput=journal
RuntimeDirectory=microfw
```

`/run` is a ramdisk, so it'll achieve the right thing.